http4s/src/test/scala/io/isomarcte/errors4s/http4s/ResponseErrorProperties.scala

```diff
@@ -0,0 +1,144 @@
+package io.isomarcte.errors4s.http4s
+
+import cats.effect._
+import cats.implicits._
+import fs2._
+import java.nio.charset.StandardCharsets
+import org.http4s._
+import org.http4s.client._
+import org.http4s.laws.discipline.arbitrary._
+import org.scalacheck.Prop.{Status => _, _}
+import org.scalacheck._
+import scodec.bits._
+
+object ResponseErrorProperties extends Properties("ResponseError") {
+  lazy val testRequest: Request[SyncIO] = Request()
+
+  lazy val invalidStatusGen: Gen[Status] = Gen
+    .choose(400, 599)
+    .filterNot(s => s === Status.Gone.code || s === Status.NotFound.code)
```

Comment:
INFO: I don't think it matters here, but I like to use `suchThat` instead of filters for scalacheck generators because that condition gets passed during test case simplification, whereas filters do not.

Reply:
TIL. I'll update that.